spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/imports/HttpImportProvider.groovy

```diff
@@ -0,0 +1,74 @@
+package org.springframework.cloud.contract.verifier.builder.imports
+
+import org.springframework.cloud.contract.verifier.config.TestFramework
+import org.springframework.cloud.contract.verifier.config.TestMode
+
+import static org.springframework.cloud.contract.verifier.config.TestFramework.CUSTOM
+import static org.springframework.cloud.contract.verifier.config.TestFramework.JUNIT
+import static org.springframework.cloud.contract.verifier.config.TestFramework.JUNIT5
+import static org.springframework.cloud.contract.verifier.config.TestFramework.SPOCK
+import static org.springframework.cloud.contract.verifier.config.TestMode.EXPLICIT
+import static org.springframework.cloud.contract.verifier.config.TestMode.JAXRSCLIENT
+import static org.springframework.cloud.contract.verifier.config.TestMode.MOCKMVC
+
+/**
+ * Provides imports based on test framework and test mode.
+ *
+ * @author Olga Maciaszek-Sharma
```

Comment:
`@since`

Reply:
`@CompileStatic`

Reply:
`@PackageScope` ?

Reply:
I would leave out `@CompileStatic`. Would have to cast between String and GString for `ImportDefinitions` constructor.